parquet-benchmarks/src/main/java/org/apache/parquet/benchmarks/ReadBenchmarks.java

```diff
@@ -18,23 +18,36 @@
  */
 package org.apache.parquet.benchmarks;
 
+import org.apache.hadoop.conf.Configuration;
 import org.apache.hadoop.fs.Path;
-import org.openjdk.jmh.annotations.Benchmark;
+import org.openjdk.jmh.annotations.*;
 import org.openjdk.jmh.infra.Blackhole;
 import org.apache.parquet.example.data.Group;
 import org.apache.parquet.hadoop.ParquetReader;
 import org.apache.parquet.hadoop.example.GroupReadSupport;
-import static org.apache.parquet.benchmarks.BenchmarkConstants.*;
 import static org.apache.parquet.benchmarks.BenchmarkFiles.*;
 
 import java.io.IOException;
 
+@State(Scope.Benchmark)
+@BenchmarkMode(Mode.AverageTime)
```

Comment:
why only average time? I think we can do `Mode.ALL` to get as many metrics as possible, which may provide useful info.